src/constants.cr

```diff
@@ -1,6 +1,11 @@
 module PassKit
-  SIGNING_CERT = ENV["SIGNING_CERT"]?
-  PRIVATE_KEY = ENV["PRIVATE_KEY"]?
-  PRIVATE_KEY_PASSWORD = ENV["PRIVATE_KEY_PASSWORD"]?
-  WWDR_CERT = ENV["WWDR_CERT"]?
+  SIGNING_CERT         = read_base64_key(ENV["SIGNING_CERT"]?)
+  PRIVATE_KEY          = read_base64_key(ENV["PRIVATE_KEY"]?)
+  PRIVATE_KEY_PASSWORD = read_base64_key(ENV["PRIVATE_KEY_PASSWORD"]?)
+  WWDR_CERT            = read_base64_key(ENV["WWDR_CERT"]?)
+
+  def read_base64_key(key : String | Nil) : String
+    raise ArgumentError.new("Base64-encoded Key is empty") unless key.presence
+    String.new(Base64.decode(key))
```

Comment:
I think making this more of a wrapper around env, where you pass the environment variable key to the method.
It will allow you to make a more helpful message. At the moment, it will just error and not detail what the source of the bad Base64 data was.

Reply:
good idea